scripts/linters/pylint_extensions.py

```diff
@@ -1519,7 +1519,6 @@ def visit_importfrom(self, node):
             node: astroid.node_classes.ImportFrom. Node for a import-from
                 statement in the AST.
         """
-
```

Comment:
Undo unexpected check*

Reply:
Sorry for that. Fixed it now!

Reply:
Ah, it's fine. I also do this kinda mistakes, I followed this process to avoid such comments from reviewers (as a tip):

After pushing changes and before assigning a reviewer, I do a self-review on my PR (like a new reviewer to a PR who is reviewing this PR for the first time.) I leave comments as a reviewer and also address those comments in a new commit. I also challenge myself to do such a review of my own PR such that no other reviewers will have any other comment on my PR. [This helps me learn (1) how to do a good review, (2) learn from another reviewer's comments about what I had missed during my self-review process, and (3) at the same time gets my PRs merged quickly!] :)

Reply:
Thanks a lot for your tips! Leaving your own comments on your PR sounds interesting! I will try out what works best for me.